src/cargo/ops/cargo_rustc/fingerprint.rs

```diff
@@ -421,6 +431,7 @@ pub fn prepare_build_cmd<'a, 'cfg>(cx: &mut Context<'a, 'cfg>, unit: &Unit<'a>)
         deps: Vec::new(),
         local: local,
         memoized_hash: Mutex::new(None),
+        rustflags: env::var("RUSTFLAGS").ok(),
```

Comment:
This is fine to leave as empty because running the build script itself (what this fingerprint is for) doesn't actually depend on `RUSTFLAGS`. Compilation of the build script should be covered by the fingerprint above I believe